message_sender/models.py

```diff
@@ -76,3 +77,25 @@ class OutboundSendFailure(models.Model):
 
     def __str__(self):  # __unicode__ on Python 2
         return str(self.id)
+
+
+@python_2_unicode_compatible
+class Channel(models.Model):
+
+    CHANNEL_TYPES = (
+        ("junebug", 'Junebug'),
```

Comment:
Maybe define each of these types as class properties and reference them from the CHANNEL_TYPES tuple?

e.g

`VUMI_TYPE = 'vumi'
JUNEBUG_TYPE = 'junebug'
CHANNEL_TYPES = ( (VUMI_TYPE, 'Vumi'), ...)`

As then you could use `object.channel_type == Channel.VUMI_TYPE` elsewhere etc.